src/main/java/xyz/nucleoid/mineout/game/MineoutActive.java

```diff
@@ -263,19 +266,20 @@ private void onPlayerComplete(long time, ServerPlayerEntity player) {
 
         this.updateSidebar();
 
-        this.gameSpace.getPlayers().sendMessage(
-                new LiteralText("")
-                        .append(player.getDisplayName())
-                        .append(" finished with a time of ")
-                        .append(new LiteralText(finishSeconds + "s").formatted(Formatting.AQUA))
-                        .append("!")
-                        .formatted(Formatting.GOLD)
-        );
+        Text message = new LiteralText("Well Done!").formatted(Formatting.GREEN);
+        Text subtitle = new LiteralText("You completed the course in ").append(String.valueOf(finishSeconds)).append("s!").formatted(Formatting.GOLD);
+
+        player.networkHandler.sendPacket(new TitleS2CPacket(10, 60, 10));
+        player.networkHandler.sendPacket(new TitleS2CPacket(TitleS2CPacket.Action.TITLE, message));
+        player.networkHandler.sendPacket(new TitleS2CPacket(TitleS2CPacket.Action.SUBTITLE, subtitle));
+
+        players.stream().filter(serverPlayerEntity -> !serverPlayerEntity.getUuid().equals(player.getUuid())).forEach(playerEntity -> {
+            playerEntity.playSound(SoundEvents.ENTITY_VILLAGER_YES, SoundCategory.PLAYERS, 1.0F, 1.0F);
+            playerEntity.sendMessage(new LiteralText("").append(player.getDisplayName()).append(" finished with a time of ").append(new LiteralText(finishSeconds + "s").formatted(Formatting.AQUA)).append("!").formatted(Formatting.GOLD), false);
```

Comment:
Can this rather be split over multiple lines as before? It is very long

Reply:
Sure